main-site/components/past-projects-section/PastProjectsSection.tsx

```diff
@@ -0,0 +1,107 @@
+  StyledPastProjectsInfoContainer,
+  StyledCamel,
+  StyledPastProjects,
+  StyledPastProjectsContainerBottom
+} from './PastProjectsSection.styles';
+import PrimaryButton from '../../../shared-ui/components/primary-button/PrimaryButton';
+const PastProjectsSection: React.FC = () => {
+  return (
+    <StyledPastProjects>
+      <StyledCamel src={Camel} />
+      <StyledFennecFox src={FennecFox} />
+      <StyledPastProjectsHeader>Past Projects</StyledPastProjectsHeader>
+      <StyledPastProjectsSection>
+        <StyledPastProjectsContainer>
+          <StyledPastProjectsPhotos src={Inky} />
+          <StyledPastProjectsInfo>
+            <StyledPastProjectsTitle>
+              Inky the Black Hole Pet, 2022
+            </StyledPastProjectsTitle>
+            <StyledPastProjectsMembers>
+              by Rachael Cheung, Euann Gu, Jessica Luo
+            </StyledPastProjectsMembers>
+            <StyledPastProjectsDescription>
+              Inky is a desktop pet! He can perform different actions: be
+              dragged around, sleep, eat, etc.
+            </StyledPastProjectsDescription>
+            <PrimaryButton
+              btnText="View Project"
+              btnLink="https://devpost.com/software/inky-the-black-hole-pet
+              "
+              newTab
+            />
+          </StyledPastProjectsInfo>
+        </StyledPastProjectsContainer>
+
+        <StyledPastProjectsContainer>
+          <StyledPastProjectsPhotos src={Swaple} />
+          <StyledPastProjectsInfo>
+            <StyledPastProjectsTitle>Swaple, 2022</StyledPastProjectsTitle>
+            <StyledPastProjectsMembers>
+              by Bob Zheng, Tze Zhe Brandon Lim, Nathan Kim, Daniel Ostapenko
+            </StyledPastProjectsMembers>
+            <StyledPastProjectsDescription>
+              A parody of wordle, but you swap letters from a list of words
+              instead! Try and complete the game in as few moves as possible.
+            </StyledPastProjectsDescription>
+            <PrimaryButton
+              btnText="View Project"
+              btnLink="https://brandonlim8890.github.io/wordle_variation/"
+              newTab
+            />
+          </StyledPastProjectsInfo>
+        </StyledPastProjectsContainer>
+
+        <StyledPastProjectsContainerBottom>
+          <StyledPastProjectsPhotos src={Duck} />
+          <StyledPastProjectsInfo>
+            <StyledPastProjectsTitle>Oh Duck!, 2022</StyledPastProjectsTitle>
+            <StyledPastProjectsMembers>by Karyna Yen</StyledPastProjectsMembers>
+            <StyledPastProjectsDescription>
+              'Oh, Duck!' is a chrome chrome extension that replaces profane
+              words with a duck emoji.
+            </StyledPastProjectsDescription>
+            <PrimaryButton
+              btnText="View Project"
+              btnLink="https://www.google.com/url?q=https://devpost.com/software/oh-duck&sa=D&source=editors&ust=1666054462827053&usg=AOvVaw1GNRfexvIUKuYxcp3Ey09w"
```

Comment:
Clicking this link takes me here
<img width="1498" alt="Screen Shot 2022-10-18 at 3 41 58 PM" src="https://user-images.githubusercontent.com/59738880/196528242-f56637d1-4c1b-48b7-b81f-c5e2ad4e66b2.png">

Link this instead https://devpost.com/software/oh-duck